winterfell/webapp/src/js/app.js

```diff
@@ -96,7 +96,22 @@ app.config(['$stateProvider', '$urlRouterProvider', function ($stateProvider, $u
     name: 'root.claimselect',
     url: '/claim/{claimId}/select-forms',
     templateUrl: 'templates/claimSelectForms.html',
-    controller: 'claimSelectFormsCtrl'
+    controller: 'claimSelectFormsCtrl',
+    resolve: {
+      forms: ['net', '$q', '$stateParams', function(net, $q, $stateParams) {
+        var deferred = $q.defer();
+
+        net.getFormsForClaim($stateParams.claimId).then(
```

Comment:
just a question. Why do this here in this resolve? As opposed to calling it when the controller starts?